api-wrapper/build_run_local.sh

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (c) 2022 SAP SE (Catena-X Consortium)
+#
+# See the AUTHORS file(s) distributed with this work for additional
+# information regarding authorship.
+#
+# See the LICENSE file(s) distributed with this work for
+# additional information regarding license terms.
+#
+
+#
+# Shell script to build and run Catena-X@Home on your local container environment.
+#
+# Prerequisites:
+#   Windows, (git)-bash shell, java 11 (java) and maven (mvn) in the $PATH.
+#   Unix, zsh, java 11 (java) and maven (mvn) in the $PATH.
+#
+# Synposis:
+#   ./build_run_local.sh
+#
+# Comments:
+#   
+#
+
+export MAVEN_OPTS=(${GRADLE_PROPS[*]})
+
+# build semantics
+cd services/semantics
+mvn clean install -DskipTests
+# build api-wrapper
+cd ../api-wrapper
+chmod +x gradlew
+./gradlew ${MAVEN_OPTS} clean build
+# build backend-data-service
+cd ../backend-data-service
+chmod +x gradlew
+./gradlew ${MAVEN_OPTS} clean build
+# build control-plane
+cd ../control-plane
+chmod +x gradlew
+./gradlew ${MAVEN_OPTS} clean build
+# build data-plane
+cd ../data-plane
+chmod +x gradlew
+./gradlew ${MAVEN_OPTS} clean build
+# change directory to root 
+cd ../..
+
+# 
+export DOCKER_PLATFORM=${DOCKER_PLATFORM:-linux/amd64}
+
+# build docker 
+docker-compose --file docker-compose.yml --project-name catenax_at_home --verbose build
+docker-compose --file docker-compose.yml --project-name catenax_at_home --verbose up
```

Comment:
- The `--file` flag is not needed.
- The `--verbose` flat also.
- Setting the `--project-name` flag is a great idea! But I would then set in the `docker-compose.yml` files directly. Can you do that?

```suggestion
docker-compose build
docker-compose up
```

Reply:
The `--file` is there for some other reasons. I will work with multiple compose files, therefore i need this one. By now only the default compose-file is referenced here.

I will integrate the project-name in the docker-compose for a better overview of the container naming.